Source/WTF/wtf/PlatformEnable.h

```diff
@@ -890,7 +890,8 @@
 #endif
 #endif
 
-#if !defined(ENABLE_PDFJS) && (PLATFORM(COCOA) || PLATFORM(GTK) || PLATFORM(WPE))
+// FIXME: Reenable PDFJS by default for Cocoa (https://bugs.webkit.org/show_bug.cgi?id=242263).
```

Comment:
Nit: Could you file a radar (or Bugzilla) for this task so we can be sure to re-enable it here?